internal/devicescale/impl_desktop_unix.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2018 The Ebiten Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build (dragonfly || freebsd || linux || netbsd || openbsd || solaris) && !android && !ios && !js
+// +build dragonfly freebsd linux netbsd openbsd solaris
+// +build !android
+// +build !ios
+// +build !js
+
+package devicescale
+
+import (
+	"math"
+
+	"github.com/jezek/xgb"
+	"github.com/jezek/xgb/randr"
+	"github.com/jezek/xgb/xproto"
+)
+
+func impl(x, y int) float64 {
+	// BEWARE: if https://github.com/glfw/glfw/issues/1961 gets fixed, this function may need revising.
+	// In case GLFW decides to switch to returning logical pixels, we can just return 1.0.
+
+	// Note: GLFW currently returns physical pixel sizes,
+	// but we need to predict the window system-side size of the fullscreen window
+	// for our `ScreenSizeInFullscreen` public API.
+	// Also at the moment we need this prior to switching to fullscreen, but that might be replacable.
+	// So this function computes the ratio of physical per logical pixels.
+	m := monitorAt(x, y)
+	sx, _ := m.GetContentScale()
+	monitorX, monitorY := m.GetPos()
+	xconn, err := xgb.NewConn()
+	defer xconn.Close()
+	if err != nil {
+		// No X11 connection?
+		// Assume we're on pure Wayland then.
+		// GLFW/Wayland shouldn't be having this issue.
+		// log.Print("No X11.")
```

Comment:
`log.Print` comments seem redundant so far. As Ebiten doesn't log anything (at least now), please remove them.

Reply:
Removed them all; I had used them for debugging and thought they may be useful if debugging it again.